rabbitmq/README.md

```diff
@@ -76,6 +76,7 @@ Configuration Options:
 | Option                           | Required | Description                                                                                                                                                                                                                                                                                                                                                                                                 |
 |----------------------------------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `rabbitmq_api_url`               | Yes      | Points to the API url of the [RabbitMQ Managment Plugin][9].                                                                                                                                                                                                                                                                                                                                                |
+| `ssl_verify`               | No      |  Set to False to skip verification of ssl cert chain when the `rabbitmq_api_url` uses https. The default is True                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
i added it here because, though it's not a very important setting in that most people will probably not set it, it really pertains mostly to the api_url and whether it uses http or https.

Reply:
Should it be `tls_verify`?

Reply:
hmm the original pr had `ssl_verify` and it seems to work for me with that option. also the code appears to be looking for ssl_verify

`ssl_verify = is_affirmative(instance.get('ssl_verify', True))`

Reply:
Hallo, sorry. [The change](https://github.com/DataDog/integrations-core/commit/9733571b02a7fa667260553868dc5bc27a8f4edc#diff-a66019d9ec2378724d26a89639133921) happened only a few days ago. We are standardizing all request related config options across integrations. The new standard config option is `tls_verify`.  `ssl_verify` will continue to work thanks to a [remapper](https://github.com/DataDog/integrations-core/blob/master/rabbitmq/datadog_checks/rabbitmq/rabbitmq.py#L145)

Reply:
cool. no problem. easy change to make.